<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="Create_CH_Lights_groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Create CH Lights</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>21</y>
      <width>99</width>
      <height>186</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="Key_Light_checkBox">
         <property name="text">
          <string>Key_Light</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="Fill_Light_checkBox">
         <property name="text">
          <string>Fill_Light</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="Rim_L_Light_checkBox">
         <property name="text">
          <string>Rim_L_Light</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="Rim_R_Light_checkBox">
         <property name="text">
          <string>Rim_R_Light</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="Dome_Light_checkBox">
         <property name="text">
          <string>Dome_Light</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="Select_All_checkBox">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Select All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Create_Light_pushButton">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Create Light</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
